src/ui/server/controllers/insurance/business/turnover/index.ts

```diff
@@ -0,0 +1,71 @@
+import { PAGES } from '../../../../content-strings';
+import { Request, Response } from '../../../../../types';
+import { TEMPLATES, ROUTES, FIELD_IDS } from '../../../../constants';
+import { FIELDS } from '../../../../content-strings/fields/insurance/your-business';
+import insuranceCorePageVariables from '../../../../helpers/page-variables/core/insurance';
+import chooseDateFormat from '../../../../helpers/date/choose-date-format';
+
+const { EXPORTER_BUSINESS } = FIELD_IDS.INSURANCE;
+const { FINANCIAL_YEAR_END_DATE } = EXPORTER_BUSINESS.TURNOVER;
+
+const { TURNOVER } = PAGES.INSURANCE.EXPORTER_BUSINESS;
+const { TURNOVER: TURNOVER_TEMPLATE } = TEMPLATES.INSURANCE.EXPORTER_BUSINESS;
+
+export const TEMPLATE = TURNOVER_TEMPLATE;
+
+const { INSURANCE_ROOT, EXPORTER_BUSINESS: EXPORTER_BUSINESS_ROUTES } = ROUTES.INSURANCE;
+
+const { TURNOVER_ROOT } = EXPORTER_BUSINESS_ROUTES;
+
+const { TURNOVER: TURNOVER_FIELDS } = FIELDS;
+
+const pageVariables = (referenceNumber: number) => ({
+  FIELDS: {
+    FINANCIAL_YEAR_END_DATE: {
+      ID: FINANCIAL_YEAR_END_DATE,
+      ...TURNOVER_FIELDS[FINANCIAL_YEAR_END_DATE],
+    },
+  },
+  POST_ROUTES: {
+    NATURE_OF_BUSINESS: `${INSURANCE_ROOT}/${referenceNumber}${TURNOVER_ROOT}`,
```

Comment:
I don't think we need this? Only `SAVE_AND_BACK_URL`? if it's just save and back (which I think is the case) we don't need `POST_ROUTES` either

Reply:
I have followed above for the all the other pages in your-business? Forgot to rename NATURE_OF_BUSINESS to TURNOVER

Reply:
We only need it for companies house page because there are 2 forms - also noticed there is this in the "continue" submit button:

```js
'formaction': POST_ROUTES.NATURE_OF_BUSINESS
```

We don't need this, native HTML will pick up the same route without this, simpler 😄